Documentation/users/ldap-integration.md

```diff
@@ -0,0 +1,125 @@
+# LDAP integration
+
+Tectonic Identity authenticates clients, such as `kubectl` and Tectonic Console, for access to the Kubernetes API and, through it, to Tectonic cluster services. All Tectonic clusters use [Role-Based Access Control (RBAC)][rbac-config] to govern access to cluster services. Tectonic Identity authenticates a user's identity, and RBAC enforces authorization based on that identity. Tectonic Identity can map cluster RBAC bindings to an existing LDAP (Lightweight Directory Access Protocol) system over a secure channel.
+
+Configure Tectonic Identity to map to your existing LDAP service, to enable user authentication for defined users and groups.
+
+## Configuring Tectonic Identity for LDAP authentication
+
+Tectonic Identity is configured through the Tectonic Console to allow for LDAP user authentication. The following information is required to integrate with an LDAP server:
+
+* The LDAP server host name and port
+* An LDAP service account capable of querying for users and groups
+* An LDAP base distinguished name (dn), from which the search for users and groups will begin
+* Attributes used to describe users and groups. For example: email and username
+* (Optional) The root CA if certificate CA verification is desired
+
+Use Tectonic Console to enable LDAP authentication in your cluster:
+
+1. Go to *Administration > Cluster Settings*, and click *LDAP*
+
+2. In the LDAP window that opens, enter your host and the (optional) port of the LDAP server in the form `host:port`, and select a verification option:
+
+ * **No SSL:** required if your LDAP host does *not* use SSL.
+ * **Skip verification:** select to skip verification of the CA.
+ * **Root CA:** if selected, enter the PEM data containing the root CAs.
+
+3. Click *Continue* to enter your LDAP Service Account username and password.
+
+question - is this a 'service account' as defined for Tectonic clusters? if so, I want to point readers to creating-service-accounts.md - beth
+
+4. Click *Continue* to enter user search criteria:
+
+  * **Base DN**: The root LDAP directory to begin the user search. Translates to the query: `(&(objectClass=person)(uid=<username>))`.
+  * **Filter**: Filter(s) applied to every user search to limit the results. For example, when a user search executes, object results could be limited to those with an `objectClass` of [person][person-ldap].
+  * **Username Attribute**: The end user's username for login. The field used when searching for users. This is the field users will use to login. For example, a commonName (assuming it's unique) or an email address.
+  question - elsewhere in the doc, RP specified that Tectonic requires that this field be email. is that true? - beth
```

Comment:
For some obscure reasons, only email attribute will be used as the user's name when talking to Kubernetes. The hack to get around this is to set `Email Attribute` to something that isn't the user's email.

Reply:
so, from your example below, it looks like the confusing bit is the "naming of things" in Tectonic Identity - in that the TI "User Attribute" maps to the LDAP `dn`, correct? I'll see if I can't edit too 1) add more specific mapping information, and 2) swap in the schema you list below as the example.